importScripts/sql/transformations/LocationHistoryExportView.sql

```diff
@@ -1,133 +1,56 @@
-    ('Museum Storeroom', 'Museum Store'),
-    ('Storeroom ,', 'Museum Store'),
-    ('Storeroom,', 'Museum Store'),
-    ('Storeroom', 'Museum Store'),
-    ('Museum Room ,', 'Museum Store'),
-    ('Museum Room,', 'Museum Store'),
-    ('Museum Room', 'Museum Store'),
-    ('Museum Store ,', 'Museum Store'),
-    ('Museum Store,', 'Museum Store'),
-    ('Museum Store', 'Museum Store'),
-    ('Museum ,', 'Museum Store'),
-    ('Museum,', 'Museum Store'),
-    ('Museum', 'Museum Store'),
-    ('Passage ,', 'Passage'),
-    ('Passage ,', 'Passage'),
-    ('Passage,', 'Passage'),
-    ('Passage', 'Passage'),
-    ('Foyer, ', 'Foyer'),
-    ('Foyer', 'Foyer'),
-    ('Meeting room ,', 'Meeting room'),
-    ('Meeting room ,', 'Meeting room'),
-    ('Meeting room,', 'Meeting room'),
-    ('Meeting room', 'Meeting room'),
-    ('Tranby room ,', 'Tranby room'),
-    ('Tranby room,', 'Tranby room'),
-    ('Tranby room', 'Tranby room'),
-    ('T.R.,', 'Tranby room'),
-    ('T.R.', 'Tranby room'),
-    ('Container ,', 'Container'),
-    ('Container,', 'Container'),
-    ('Container', 'Container'),
-    ('Locked Storeroom ,', 'Locked Storeroom'),
-    ('Locked Storeroom,', 'Locked Storeroom'),
-    ('Locked Storeroom', 'Locked Storeroom'),
-    ('Library ,', 'Library'),
-    ('Library,', 'Library'),
-    ('Library', 'Library');
-
-
 CREATE OR REPLACE VIEW LocationHistoryExport AS
 SELECT
-    NULLIF(lh.LocationHistoryID, '')       AS LocationHistoryID,
-    NULLIF(lh.AuthorizedBy, '')            AS AuthorizedBy,
-    NULLIF(lh.Location, '')                AS Location,
-    NULLIF(lh.LocationDate, '')            AS LocationDate,
-    NULLIF(lh.LocationStatus, '')          AS LocationStatus,
-    NULLIF(lh.PrimaryKey_Object_Table, '') AS PrimaryKey_Object_Table,
-    NULLIF(o.Accession_Full_ID, '')        AS o_Accession_Full_ID,
-    NULLIF(o.ItemType, '')                 AS o_ItemType,
-    CASE
-    WHEN
-        # The location has no parent, and will appear in `childStorageLocation` below
-        TRIM(lh.Location) = sl.Alias
-        THEN NULL
-    ELSE
-        # Otherwise we fill in from the standardised name
-        sl.StandardisedName
-    END                                    AS topLevelLocation,
+    NULLIF(TRIM(lh.LocationHistoryID), '')       AS LocationHistoryID,
+    NULLIF(TRIM(lh.AuthorizedBy), '')            AS AuthorizedBy,
+    NULLIF(TRIM(lh.Location), '')                AS Location,
+    NULLIF(TRIM(lh.LocationDate), '')            AS LocationDate,
+    NULLIF(TRIM(lh.LocationStatus), '')          AS LocationStatus,
+    NULLIF(TRIM(lh.PrimaryKey_Object_Table), '') AS PrimaryKey_Object_Table,
+    NULLIF(TRIM(o.Accession_Full_ID), '')        AS o_Accession_Full_ID,
+    NULLIF(
+        LOWER(REPLACE(TRIM(o.ItemType), ' ', ''))
+        , ''
+    )                                            AS o_ItemType,
+    NULLIF(TRIM(lhc.Room), '')                   AS Room,
+    NULLIF(TRIM(lhc.ShelfDrawer), '')            AS ShelfDrawer,
```

Comment:
The following two blocks copy ShelfDrawer into two columns, `Shelf` and `Drawer` depending on whether it has 'drawer' in it.